data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+			`	"Can you take us to Mr. Foster's museum? We'll take the cube and fill him in on the details."`
+			`	You agree, and after they're all ready, you make your way out to the dig site. Surprisingly, the three Hai don't get many weird looks if they walk a bit further behind you. You arrive at the museum and they immediately break character by being very obviously excited to see the exhibits. This probably comes as a relief to onlookers, however, who would normally expect that type of kid to vandalize priceless artifacts.`
+			`	Finally, you arrive at Albert's workshop. He stops brushing what looks like what was once a television and looks startled at you and the three "kids". The Hai take off their hood and bandanna, causing his jaw to nearly drop to the floor.`
+			`	"I'll be damned. The Hai really exist!" He exclaims.`
+			`	"Of course we do" Eyama laughs. "Haven't you heard all the stories?"`
+			`	"I had my doubts. Merchants are well known for telling tall tales." He smiles. "Anyway, I'm Albert Foster, archaeologist, pleased to meet you all."`
+			`	They introduce themselves, and very soon the conversation evolves to a kind of jargon even Albert has trouble following. They talk about the box, and their findings, while Albert talks about the history uncovered in the dig site, and insights on the Sheragi daily life and culture. You close the door to make sure nobody else is startled.`
+			`	"We've managed to open the box and we're making a blueprint of it," says Desyo. "Once that's done, we might be able to rebuild the broken pieces and recover the data. We already have documentation on how their hardware worked."`
+			`	"The catch is that this is a collaboration now. We'll help you, if you share your findings with us," Rakei continues.`
+			`	"Sounds like a win-win to me. We'll arrange the details later. <first>, thank you so much for your help. We'll continue studying this artifact, and we'll send you a message when we're done."`
+			`	You say goodbye to Albert, Rakei, Eyama, and Desyo, and make your way back to your ship.` 
+		event "cube decoded" 40 50
+		
+mission "The Emerald Sword 1"
+	description "Visit Albert Foster on Zug."
+	landing
+	name "Black box decoded"
+	source
+		near "Zubeneschamali" 1 100
+	destination "Zug"
+	
+	to offer
+		has "event: cube decoded"
+	
+	on offer
+		conversation
+			`You're still in the middle of the post-flight check when you receive a new message from Albert Foster.`
+			`	"Hello <first>. I have some good news! We have decoded what's in the black box. You should come visit us on Zug; we have plenty to tell you!"`
+				accept
+
+mission "The Emerald Sword 2"
+	landing
+	description "Bring an additional Jump Drive to Zug."
+	name "The Emerald Sword"
+	source "Zug"
+	destination "Zug"
+	
+	to offer
+		has "event: cube decoded"
+		has "The Emerald Sword 1: done"
+	
+	on offer
+		require "Jump Drive"
```

Comment:
Should i remove the Jump Drive requirement? I don't think a "blocked" text would fit in here, but this mission is post-main-quest, so the player should theoretically have a JD. Maybe i'll make the question afterwards a choice

Reply:
It can be pretty reasonably assumed that the player does have a JD at this point, but we can't really guarantee that. As such, it's either a matter of removing this requirement, or adding this requirement to the previous mission.

Reply:
I changed the choice to fix this problem.